styles/styles.css

```diff
@@ -0,0 +1,662 @@
+@import url("./fonts.css");
+@import url("./all.min.css");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  max-width: 1264px;
+  padding: 0 16px;
+  margin: 0 auto;
+}
+
+input {
+  border: none;
+}
+
+.phone {
+  display: none;
+}
+
+.trip__background {
+  background-image: url("../images/castelmezzano-1979546 1.jpg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.trip__contanier {
+  max-width: 1264px;
+  padding: 0 16px;
+  margin: 0 auto;
+}
+
+.trip__header {
+  padding-top: 51px;
+  padding-bottom: 202px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.trip-header__left-svg {
+  width: 205px;
+  height: 40px;
+}
+
+.trip-header__right {
+  display: flex;
+  flex-direction: column;
+}
+
+.right__under {
+  display: flex;
+}
+
+.under__href {
+  margin-right: 24px;
+  font-size: 24px;
+  line-height: 28px;
+  justify-content: space-around;
+  color: #ffffff;
+  text-decoration: none;
+}
+
+.under__logotip {
+  margin-left: 24px;
+  justify-content: space-around;
+  gap: 24px;
+}
+
+.under-logotip__night {
+  width: 30px;
+  height: 30px;
+  margin-right: 24px;
+}
+
+.under-logotip__account {
+  width: 40px;
+  height: 40px;
+}
+
+.under-logotip__menu {
+  display: none;
+}
+
+.under__line {
+  width: 50px;
+  background-color: #f5bd41;
+  border: 2px solid #f5bd41;
+}
+
+.trip__page {
+  padding-bottom: 120px;
+}
+
+.trip-page__header {
+  color: #ffffff;
+  font-weight: 500;
+  font-size: 50px;
+  margin-bottom: 100px;
+}
+
+.trip-page__underform {
+  padding-left: 24px;
+  display: flex;
+  gap: 221px;
+  color: #ffffff;
+  font-size: 18px;
+  line-height: 21px;
+  margin-bottom: 24px;
+}
+
+.trip-page__form {
+  display: flex;
+  margin-right: 9px;
+  margin-bottom: 120px;
+  background: #FFFFFF;
+  border: 3px solid #F3F3F4;
+  border-right: #3077c6;
+  border-radius: 8px;
+}
+
+.form__filters {
+  display: flex;
+}
+
+.form__city-desktop {
+  width: 467px;
+  font-weight: 400;
+  font-size: 18px;
+  color: #383838;
+  padding-left: 24px;
+}
+
+.page-form__city-phone-img {
+  display: none;
+}
+
+.search-city-phone {
+  display: none;
+}
+
+.form__city-phone {
+  display: none;
+}
+
+.page-form__data {
+  width: 259px;
+  font-weight: 400;
+  font-size: 18px;
+  color: #383838;
+  text-align: center;
+  border: 3px solid #F5BD41;
+  border-radius: 8px;
+}
+
+.page-form__nomer-phone {
```

Comment:
Try not to use russian transliterations in your code.

Reply:
ok